.github/workflows/command-robotpy-pr.yml

```diff
@@ -0,0 +1,23 @@
+name: Comment on PR for robotpy
+
+on:
+  pull_request:
+    types:
+      - opened
+    paths:
+      - 'wpilibNewCommands/**'
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Comment on PR
+        uses: actions/github-script@v7
+        with:
+          script: |
+            github.rest.issues.createComment({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                body: 'This PR modifies commands. Please open a corresponding PR for [Python Commands](https://github.com/robotpy/robotpy-commands-v2/)'
```

Comment:
... or comment with a justification why that isn't necessary. Also should ask the user to link to the PR.